### PR TITLE
update crannogmen to v1.1

### LINKED ISSUE #111: Update Crannogmen to Ver. 1.1
Update the following card from:
  > Crannogmen (Ver. 1.0) 
> 4 Cost | Non-Loyal [stark] Character
> 4 STR  | [military] [intrigue] 
> ***Army. House Reed.***
> 
> Shadow (3). Stealth.
> **Reaction:** After you win a challenge in which Crannogmen is participating, if the losing opponent has a higher initiative value than you, return Crannogmen to shadows to have that player choose and kill a participating character they control.

  to

  > Crannogmen (Ver. 1.1) 
> 4 Cost | Loyal [stark] Character
> 4 STR  | [military] [intrigue] 
> ***Army. House Reed.***
> 
> Shadow (3). Stealth.
> **Reaction:** After a ***House Reed*** card enters play, place a poison token on a character controlled by an opponent. Then, if the number of poison tokens are equal or higher than that characters STR, return Crannogmen to shadows to kill it.

  General steps
  - [ ] Update the code for this card (within 'server/game/cards/{current pack}')
  - [ ] (Optional) Test the implementation locally
  - [ ] Push changes to develop branch & close this issue
  
  Alternatively, it will also be automatically closed once this card is marked as "Implemented" on the spreadsheet.

### PR DESCRIPTION
**Pull request by EvadLhorg**
closes #111

### CODE REVIEW

File: server/game/cards/23-AHaH/Crannogmen.js [hunk 1]
```diff
@@ -1,26 +1,34 @@
...
-                    GameActions.kill({ card: context.target, player: context.choosingPlayer }),
+                    GameActions.placeToken({
+                        card: context.target, token: Tokens.poison
+                    }).then({
+                        condition: () => context.target.getStrength() <= context.target.tokens[Tokens.poison],
+                        cost: ability.costs.putSelfIntoShadows(),
+                        message: {
+                            format: 'Then {player} uses {source} to kill {poisonedCharacter}',
+                            args: { poisonedCharacter: () => context.target }
+                        },
```
Comment (Patane97) [inline server/game/cards/23-AHaH/Crannogmen.js:27]:
```suggestion
                        message: 'Then, {player} returns {source} to shadows to kill {target}',
```

File: server/game/cards/23-AHaH/Crannogmen.js [hunk 2]
```diff
@@ -1,26 +1,34 @@
...
-            },
-            message: {
-                format: '{player} uses {source} to have {loser} choose and kill {target}',
-                args: { loser: context => context.event.challenge.loser }
+                type: 'select',
+                cardCondition: card => card.location === 'play area' && card.controller !== this.controller &&
+                                       card.getType() === 'character'
             },
             handler: context => {
+                this.game.addMessage('{0} uses {1} to place a Poison token on {2}', context.player, this, context.target);
```
Comment (Patane97) [inline server/game/cards/23-AHaH/Crannogmen.js:17]: Probably best to have this as an ability message (outside of the handler):
```
message: '{player} uses {source} to place a Poison token on {target}'
```

**Review by Patane97 — APPROVED** [review]
Just some minor adjustments I'd suggest for the messages; looks like it would work fine (haven't actually tested it though, hopefully you have 😄).

Will approve, just suggest making those message changes before merging!

File: server/game/cards/23-AHaH/Crannogmen.js (hunk 1 above)
Comment (EvadLhorg) [reply server/game/cards/23-AHaH/Crannogmen.js]: {target} can´t be used here as it is not set in the context of the then part, so i left the args part in

**Pull request merged**